cmake/FindWayland.cmake

```diff
@@ -2,10 +2,10 @@
 #
 # This will define:
 #
-#   WAYLAND_FOUND       - True if Wayland is found
-#   WAYLAND_LIBRARIES   - Link these to use Wayland
-#   WAYLAND_INCLUDE_DIR - Include directory for Wayland
-#   WAYLAND_DEFINITIONS - Compiler flags for using Wayland
+#   Wayland_FOUND       - True if Wayland is found
+#   Wayland_LIBRARIES   - Link these to use Wayland
+#   Wayland_INCLUDE_DIR - Include directory for Wayland
+#   Wayland_DEFINITIONS - Compiler flags for using Wayland
```

Comment:
If you delete `Wayland_LIBRARIES` and `Wayland_INCLUDE_DIR`, edit this accordingly

And document the imported targets we now make, e.g.
```
# Wayland::Client     - An imported target for Wayland Client
etc
```